components/quick_input.jsx

```diff
@@ -95,6 +95,7 @@ export default class QuickInput extends React.PureComponent {
         if (this.props.onClear) {
             this.props.onClear();
         }
+        this.focus();
```

Comment:
Looks fine to me but could you link on how we caused this regression?

Reply:
i don't think it ever had its focus back, at somepoint it didn't have the X button to clear, so it probably wasn't needed. we later added the x, but never thought on focus.